test/index.test.js

```diff
@@ -1,3 +1,28 @@
-test('Initial test', () => {
-  expect(1).toBe(1);
+const request = require('supertest');
+const connection = require('../server/database/config/connection');
+const dbBuild = require('../server/database/config/build');
+
+const app = require('../server/app');
+
+beforeAll(() => {
+  return dbBuild();
+});
+
+afterAll(() => {
+  return connection.end();
+});
+
+describe('Admin, (/cohorts/:cohortId)', () => {
+  test('Route /cohorts/1 status 200, json header, data.message = Cohort deleted successfully ', (done) => {
+    return request(app)
+      .delete('/api/v1/cohorts/1')
+      .expect(200)
+      .expect('Content-Type', /json/)
+      .end((err, res) => {
+        const { data } = res.body;
+        if (err) return done(err);
+        expect(data.message).toBe('Cohort deleted successfully');
```

Comment:
@MohammedAlghazali This test looks good.  Another thing you could add, to ensure that the delete endpoint is actually deleting things, is to query the database and ensure that cohort 1 really has been removed.